autoload/clap/themes.vim

```diff
@@ -162,6 +162,8 @@ function! s:reverse_PopupCursor() abort
 endfunction
 
 function! s:init_theme() abort
+  hi ClapShadow guibg=#000000
```

Comment:
```suggestion
+  hi ClapDefaultShadow guibg=#000000
+  hi default link ClapShadow ClapDefaultShadow
```

So that user can easily override the highlight in their vimrc.